appConfig/appConfig.yaml

```diff
@@ -5,11 +5,11 @@
 - key: PAYMENT_TOPIC_ADDRESS
   value: "ffc-demo-payment"
 - key: PAYMENT_SUBSCRIPTION_ADDRESS
-  value: "payment"
+  value: "ffc-demo-payment"
 - key: SCHEDULE_TOPIC_ADDRESS
   value: "ffc-demo-schedule" 
 - key: SCHEDULE_SUBSCRIPTION_ADDRESS
-  value: "payment"  
+  value: "ffc-demo-payment"  
```

Comment:
Should be : ffc-demo-schedule-payment

Reply:
@rajeshkaremane : No, format is **namespace-topicsubscriptionname** . so above is right.

Reply:
Should be a separate ticket then